frontend/src/pages/VacationPlanner.tsx

```diff
@@ -112,23 +132,40 @@ export const VacationPlanner = () => {
       }
 
       // So that loading times are shorter, we only take one
-      let sliced_users = group_users.slice(-1);
+      let sliced_users = group_users.slice(-12);
+
+      let vacationRows: [] = [];
+      vacationRows.push([
+        { type: "string", id: "User" },
+        { type: "string", role: "tooltip", p: { html: true } },
+        { type: "date", id: "Start" },
+        { type: "date", id: "End" },
+      ]);
 
-      const vacation_entries: FetchedTimeEntry[] = [];
       for await (let group of sliced_users) {
         let entries = await getVacationTimeEntries(
-          new Date(2022),
-          new Date(2023),
+          new Date(2021, 11, 20),
+          new Date(2021, 12, 30),
           group.toString()
         );
-        vacation_entries.push(...entries);
+        entries.map((entry) => {
+          entry.group = groupId;
+        });
+        entries.map((entry: FetchedTimeEntry) => {
+          vacationRows.push([
+            entry.user.name ? entry.user.name : entry.user,
+            "test",
+            new Date(entry.spent_on),
+            new Date(entry.spent_on).getTime() + 86400000,
```

Comment:
This is one day in milliseconds, right? A short comment would be helpful, I had to calculate :)